src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
+            'by an algorithm. Note that this should be set to false if you',
```

Comment:
What would be _really_ helpful is if we use some delimiter to mark things as linkable attributes so that in the docs, users can click on them and go to their definition. E.g.

`description: 'This sets some thing, provided that [context.attribute](toplevel.midlevel.bottomlevel) is true.'`

I'm not sure what the process of `attributes.js` files -> docs is, but I'd think we could make it happen.